pg_search/src/postgres/insert.rs

```diff
@@ -46,7 +61,19 @@ pub unsafe extern "C" fn aminsert(
     _check_unique: pg_sys::IndexUniqueCheck,
     _index_info: *mut pg_sys::IndexInfo,
 ) -> bool {
-    aminsert_internal(index_relation, values, isnull, heap_tid)
+    let relid = pg_sys::RelationGetRelid(index_relation);
```

Comment:
The `oid` function of `PgRelation` can be used to abstract away this `pg_sys` call. (I believe this function was renamed in pg12 and 13).